mne/epochs.py

```diff
@@ -37,6 +37,7 @@
 from .channels.channels import (ContainsMixin, UpdateChannelsMixin,
                                 SetChannelsMixin, InterpolationMixin)
 from .filter import resample, detrend, FilterMixin
+from .decoding.mixin import VectorizerMixin
```

Comment:
you get rid of circular imports by moving things around or using nested
imports :)